gatsby-config.js

```diff
@@ -19,8 +19,8 @@ if ((metaConfig === null) || (metaConfig === undefined)) {
 
 module.exports = {
   adapter: adapter({
-    excludeDatastoreFromEngineFunction: false,
-    imageCDN: false,
+    excludeDatastoreFromEngineFunction: true,
```

Comment:
```suggestion
    excludeDatastoreFromEngineFunction: process.env.GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE || true,
```

Reply:
This may be unnecessary. It looks like the plugin has two ways of setting this to true (either by setting the excludeDatastoreFromEngineFunction to true OR by setting the variable GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE to true). I cannot tell if one overrides the other (so if someone were to set GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE as false, would that override this code?)

I will try and test it. And if it automatically overrides it, then we can just set it to true, knowing we can use the GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE to override it.

https://www.gatsbyjs.com/plugins/gatsby-adapter-netlify/

Reply:
After testing this on a throwaway branch, the time didn't actually change (it was still 6 minutes, even when I set the variables to false). I'm going to remove this change request because the gatsby-adapter-netlify plugin seems to have taken both the variable and the option into account.

Reply:
Confirmed. It looks like the GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE does not override the option value if both exist. The option value will take precedence: https://github.com/gatsbyjs/gatsby/blob/c91ed287fd319a345c2f27877e20656826767e92/packages/gatsby-adapter-netlify/src/index.ts#L126